web/Dockerfile

```diff
@@ -1,12 +1,9 @@
-FROM cypress/base:14
+FROM node:14
 
-RUN npm i -g npm@^7
+RUN npm i -g npm@^6
```

Comment:
why npm 6 and not npm 7?

Reply:
I reverted the whole thing to an older version.  I can move it back to NPM 7.